DigitalLearningSolutions.Web/Styles/myAccount/notificationPreferences.scss

```diff
@@ -3,3 +3,30 @@
 .top-spaced-goback {
   margin-top: 32px;
 }
+
+.preference-list{
+  margin-bottom: 16px;
+}
+
+.preference-row {
+  border-bottom: 1px solid $color_nhsuk-grey-4;
+  margin-top: 24px;
+}
+
+.preference-row:first-child {
+  margin-top: 0;
+}
+
+.preference-tag {
+  width: 100%;
+  max-width: 10em;
+  text-align: center;
+  margin-bottom: nhsuk-spacing(4);
+}
+
+@media (min-width: 48.0625em) {

```

Comment:
Steve has used this syntax in his MR, which I think is nicer and ensures we use the NHS breakpoints (feel free to check out his MR):
@include govuk-media-query($from: tablet) {
    // Styles
  }